src/rprblender/export/world.py

```diff
@@ -247,6 +248,8 @@ def set_override(override_type):
                 override_data.image = image.name
             else:
                 override_data.color = tuple(color)
+            override_data.rotation = getattr(rpr, f'{override_type}_rotation') \
```

Comment:
Convert it to tuple to make it comparable in BackplateData, should be:
```
override_data.rotation = tuple(getattr(rpr, f'{override_type}_rotation')) \
....
```
It fixes interactive changing rotation in viewport render.